src/Pages/SingleVideo/SingleVideo.jsx

```diff
@@ -33,6 +34,7 @@ const SingleVideo = () => {
         width="100%"
         height="540rem"
         src={`https://www.youtube.com/embed/${findVideo._id}`}
+        onInferredClick={() => (token && addToHistory(findVideo))}
```

Comment:
`onInferredClick` is something new for me, have to check it out!

Reply:
Yeah, it was new for me as well